programs/ambassadors/ambassadors-program.md

```diff
@@ -0,0 +1,62 @@
+This document covers the organizational and operational aspects of the JSON Schema Ambassador Program.
+
+### What is the JSON Schema Ambassador Program?
+
+The JSON Schema Ambassador Program aims to bring JSON Schema closer to both current users and potential new project users. It also aims to provide users and community members with the necessary tools and resources to:
+
+- Promote collaboration among the JSON Schema user base.
+- Engage with the project in some way, either as a contributor, blogger, speaker, etc.
+- Speak at community events or write technical content (e.g. blog posts) that solves community needs.
+- Nurture community growth and a welcoming environment to spread knowledge about the project.
+
+### How does the program work?
+
+- The ambassador selection process will be as transparent as possible.
+- The model currently in place for adding members to the Technical Steering Committee through PR will be used.
```

Comment:
While this may be accurate for AsyncAPI, this isn't right for JSON Schema, given that (process of adding someone to the TSC) isn't active right now.

Reply:
Thanks for the heads up. I'll change it.